cpu/samd21/cpu.c

```diff
@@ -72,19 +94,35 @@ static void clk_init(void)
     while (!(SYSCTRL->PCLKSR.reg & SYSCTRL_PCLKSR_OSC8MRDY)) {}
 #endif
 
+    /* Setup GCLK2 with divider 1 (32.768kHz) */
```

Comment:
@benpicco why did you move this up? `GCLK->CTRL.reg = GCLK_CTRL_SWRST`resets what is done here, can I move it back down in #13306? Iget the same drift as before otherwise.

Reply:
I see you use it for the dfll, I'll move `GCLK->CTRL.reg = GCLK_CTRL_SWRS` up.

Reply:
Hm previously this was in the `#if CLOCK_USE_PLL` block, but a 32kHz clock is always needed so I moved it here.
I must have missed the `GCLK_CTRL_SWRS` bit.